scripts/features.sh

```diff
@@ -51,13 +51,13 @@ packages() {
 	shift
 
 	# We only allow variable names, parentheses and the operators: & | !
-	if [ "$(expr match "$cond" '.*[^A-Za-z0-9_()&|! ].*')" -gt 0 ]; then
+	if [[ "$cond" == '.*[^A-Za-z0-9_()&|! ].*' ]]; then
```

Comment:
Will this automatically match -gt 0?

Reply:
fixed. expr match will return the counted matching characters so is at leas one character like . is matched it will be greater than 0. I change it to: `grep -q ".*[^A-Za-z0-9_()&|! ].*" <<< "$cond"`